commands.md

```diff
@@ -62,31 +71,39 @@ conditionalGroup([a, b, c]);
 declare function fill(docs: Doc[]): Doc;
 ```
 
-This is an alternative type of group which behaves like text layout: it's going to add a break whenever the next element doesn't fit in the line anymore. The difference with a typical group is that it's not going to break all the separators, just the ones that are at the end of lines.
+This is an alternative type of group which behaves like text layout: it's going to add a break whenever the next element doesn't fit in the line anymore. The difference with [`group`](#group) is that it's not going to break all the separators, just the ones that are at the end of lines.
 
 ```js
-fill(["I", line, "love", line, "prettier"]);
+fill(["I", line, "love", line, "Prettier"]);
 ```
 
+Expects the `docs` argument to be an array of alternating content and line breaks. In other words, elements with odd indices must be line breaks (e.g., [`softline`](#softline)).
+
 ### ifBreak
 
 ```ts
-declare function ifBreak(ifBreak: Doc, noBreak: Doc): Doc;
+declare function ifBreak(
+  ifBreak: Doc,
+  noBreak?: Doc,
+  options?: { groupId?: symbol }
+): Doc;
 ```
 
-Prints something if the current group breaks and something else if it doesn't.
+Print something if the current `group` or the current element of `fill` breaks and something else if it doesn't.
 
 ```js
 ifBreak(";", " ");
 ```
 
+`groupId` can be used to check another _already printed_ group instead of the current group.
+
 ### breakParent
```

Comment:
```suggestion
### `breakParent`
```

etc.. since it's code words

Reply:
@lipis Won't this affect existing links to this file?

Reply:
Turns out it won't. Done.